src/map/GMap.js

```diff
@@ -258,6 +268,30 @@
         this.updateCircles();
         this.updatePins();
     };
+    
+    GMap.prototype.showStreetView = function () {
+        var map;
+        if(this.streetViewTargetSelector()){
+            map = new google.maps.Map(document.querySelector(this.streetViewTargetSelector()),{zoom: 5,mapTypeId: google.maps.MapTypeId.ROADMAP});
```

Comment:
Why do we need to create a new map?  Or is this so the street view should be hosted in a diff target? and have both active?